### PR TITLE
Create suggested conditions automatically rather than prefilling form

### PR DESCRIPTION
**Pull request by benjamineskola**
### Description of change

In the new design the conditions will be created by default and able to be removed by the user, rather than being a prefilled form that they can optionally add.

This creates them in the controller action if none exist.

### Story Link

https://trello.com/c/oFK58Hnp/3000-design-improvements-for-add-conditions

### Screenshots

Screenshot 2024-05-21 at 16 32 16

### CODE REVIEW

File: app/models/condition.rb [hunk 1]
```diff
@@ -32,6 +32,16 @@ def truncated_comment
...
   end
 
+  class << self
+    def standard_conditions
+      I18n.t(:conditions_list).map do |k, v|
+        c = Condition.new(v)
+        c.standard = true
+        c
+      end
+    end
```
Comment (pixeltrix) [inline app/models/condition.rb:42]: Isn't `standard` set to `true` in the locale file?

File: app/models/planning_application.rb [hunk 2]
```diff
@@ -671,6 +671,14 @@ def condition_set
     super || create_condition_set!
   end
 
+  def create_condition_set!
+    super
+
+    Condition.standard_conditions.map { |condition| condition.update!(condition_set: condition_set) }
+
+    condition_set
```
Comment (pixeltrix) [inline app/models/planning_application.rb:679]: Wouldn't it be better to do this in a callback on `ConditionSet`?

File: app/models/condition_set/conditions_extension.rb [hunk 3]
```diff
@@ -7,19 +7,13 @@ def sorted
     end
 
     def standard
-      standard_conditions.map do |condition|
+      Condition.standard_conditions.map do |condition|
```
Comment (pixeltrix) [inline app/models/condition_set/conditions_extension.rb:10]: Trying to work out why we don't do `select(&:standard?)` here - legacy data issue maybe?

File: app/models/condition.rb (hunk 1 above)
Comment (benjamineskola) [reply app/models/condition.rb:42]: Oh yes, seems unnecessary. Will remove.

File: app/models/condition_set/conditions_extension.rb (hunk 3 above)
Comment (benjamineskola) [reply app/models/condition_set/conditions_extension.rb:10]: I'm actually not sure if that's used at all now.

**Review by pixeltrix — APPROVED** [review]
Just a couple of minor improvements

**Pull request merged**